xarray/core/groupby.py

```diff
@@ -382,7 +382,15 @@ def fillna(self, value):
         Dataset.fillna
         DataArray.fillna
         """
-        return self._fillna(value)
+        def _yield_applied(this, other):
```

Comment:
I *think* `apply_ufunc` should even work on groupby objects, so you shouldn't need this helper function. I would be interested to know I'm wrong about that, though.